lib/views/recipe_detail_screen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import '../models/recipe.dart';
+import '../constants.dart';
+
+class RecipeDetailScreen extends StatelessWidget {
+  final Recipe recipe;
+
+  RecipeDetailScreen({required this.recipe});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(recipe.name),
+        backgroundColor: kPrimaryColor,
+      ),
+      body: SingleChildScrollView(
+        padding: kPadding,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            _buildImage(recipe.image),
+            SizedBox(height: 16),
+            _buildText(recipe.headline, kTextStyleTitle),
+            SizedBox(height: 16),
+            _buildText(recipe.description, kTextStyleDescription),
+            SizedBox(height: 16),
+            Divider(),
+            _buildInfoGrid(recipe),
+          ],
```

Comment:
It's best to create a widget using class instead of helper function so we don't need to care about performance issue when widget re-build. 

Refs:

[Widgets vs helper methods | Decoding Flutter](https://www.youtube.com/watch?v=IOyq-eTRhvo)
[More](https://steveos.medium.com/use-separate-widget-over-helper-method-in-flutter-heres-why-better-performace-433672eb7461)